File: Topic Based Problems/01 Fundamentals/1 Beginner/17 Area of a Square/Java/AreaofaSquare.java
import java.util.Scanner;

public class AreaofaSquare {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the side of a Square: ");
        double side = scanner.nextDouble();
        
        System.out.printf("Area of a Square = %.2f\n", side * side);
        
        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/1 Beginner/17 Area of a Square/C/AreaofaSquare.c
/**
 * @file AreaofaSquare.c
 * @brief Area of a Square in C
 * @version 1.0
 * @date 2023-09-24
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @copyright Copyright (c) 2023
 */

#include <stdio.h>
#include <math.h>

int main() {
    double side;
    
    printf("Enter the side of a Square: ");
    scanf("%lf", &side);

    printf("Area of a Square = %.2lf\n", side * side);

    return 0;
}
